traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -1,26 +1,82 @@
-{% extends "traits/base.html" %}
-
-{% block title %}
-Trait Detail
-{% endblock %}
+{% extends "traits/base.html" %} 
+{% block title %} Trait Detail {% endblock %}
+{% load trait_filters %} 
 
 {% block content %}
-<div class="uk-container uk-container-xlarge">
-    <h1>Trait details</h1>
-    <ul class="uk-list">
-        <li>Trait name: {{trait.name}}</li>
-        <li>Source records: {% firstof trait.number_of_source_records '0' %}</li>
-        <li>Trait status: {{trait.status}}</li>
-    </ul>
-    <h1>Mapped term details</h1>
-    {% with trait.current_mapping.term_id as term %}
-    <ul class="uk-list">
-        <li>Term: {% firstof term.curie 'None' %}</li>
-        <li>Term label: {% firstof term.label 'None' %}</li>
-        <li>Term status: {% firstof term.status 'None' %}</li>
-    </ul>
-    {% endwith %}
-    
-    
+<aside class="sidebar">
+  <div class="sidebar__wrapper">
+    <div class="sidebar__section">
+      <p class="sidebar__section-title">LAST UPDATED</p>
+      <span class="sidebar__section-text"> {{trait.timestamp_updated}} </span>
+      <hr />
+    </div>
+    <div class="sidebar__section">
+      {% if trait.current_mapping %}
+      <p class="sidebar__section-title">CURATOR</p>
+      <span uk-icon="user" class="uk-margin-small-right"></span>
+      <span class="sidebar__section-text">
+        {{trait.current_mapping.curator}}
+      </span>
+      <hr />
+      {% endif %}
+    </div>
+    <div class="sidebar__section">
+      {% if trait.current_mapping %}
+      <p class="sidebar__section-title">REVIEWED BY</p>
+      {% with trait.current_mapping.review_set.all as reviews %} 
+      {% for review in reviews %}
+      <div class="sidebar__review-section">
+        <span uk-icon="user" class="uk-margin-small-right"></span>
+        <span class="sidebar__section-text"> {{review.reviewer}} </span>
+      </div>
+      {% endfor %}
+      <hr />
+      {% endwith %}{% endif %}
+    </div>
+    <div class="sidebar__section">
+      {% if trait.current_mapping %}
+      <p class="sidebar__section-title">REVIEW STATUS</p>
+      {% if trait.current_mapping.is_reviewed %}
+      <span class="uk-label label--success"> REVIEWED</span>
+      {% else %}
+      <span class="uk-label label--warning"> IN REVIEW </span>
+      {% endif %}
+      <button class="uk-button button--success sidebar__section-review-button">
+        <span uk-icon="check"> REVIEW </span>
+      </button>
+      {% endif %}
+    </div>
+  </div>
+</aside>
+
+<div class="uk-container uk-container-xlarge trait-container">
+  <div class="info-section__wrapper">
+    <div class="info-section__column">
+      <span class="info-section__title">TRAIT NAME</span>
```

Comment:
Currently, the contents of the trait page are indented compared to the "Browse traits" button in the header:

![image](https://user-images.githubusercontent.com/10669118/85573409-c6019f00-b62d-11ea-8c11-65fc92e01a5e.png)

I'm thinking it would be better to align them all to the same left margin

Reply:
Well, the norm on headers is for them to have a smaller indent compared to the main page containers (you can look at GitHub here as an example). So my suggestion here would be to actually decrease the indent by a small amount on the header so that the distinction between itself and the container is clearer. 

Another reason for that is that in very large screens, the container will have more margin compared to smaller screens so basically the only way to align those two is to put the header links inside the same type of container

Reply:
Right, so first of all, one the second thought I do agree with you about the indents. Content having larger intent than header makes sense.

However, the problem which I noticed now is that margins changes drastically on different zoom levels:

100%
![image](https://user-images.githubusercontent.com/10669118/85893298-8502b980-b7ea-11ea-932a-8054ce6d0cf6.png)

125%
![image](https://user-images.githubusercontent.com/10669118/85893310-8e8c2180-b7ea-11ea-995b-d0d8b1ce71ac.png)

150%
![image](https://user-images.githubusercontent.com/10669118/85893326-977cf300-b7ea-11ea-9ead-2634b455ed83.png)

I think the look closest to normal is on 125% level. But it should look approximately the same at least in the 100–150% zoom range, as those are frequently used.

This can be done either in this PR, or as a folow-up ticket (perhaps united with comment from [here](https://github.com/EBIvariation/trait-curation/pull/43#discussion_r446364192))

Reply:
That's weird. I am using the `container` class that comes with UIKit but I might have to create one my own. Could you tell me the width of the screen you tested this on?

Reply:
I don't know what changed (or whether anything changed at all), but now I don't see this problem and zoom works as expected. Marking this as resolved then :)